tests/conftest.py

```diff
@@ -34,7 +34,10 @@
 
 
 try:
-    import uvloop
+    if sys.platform != "win32":
+        import uvloop
+    else:
+        import winloop as uvloop
```

Comment:
Nit: I'd do a positive check:
```suggestion
    if sys.platform == "win32":
        import winloop as uvloop
    else:
        import uvloop
```